extensions/bitwarden/src/search.tsx

```diff
@@ -229,14 +229,13 @@ function PasswordActions(props: { password: string }) {
 
 function UsernameActions(props: { username: string }) {
   return (
-    <ActionPanel.Item
-      shortcut={{ modifiers: ["cmd"], key: "u" }}
+    <Action.CopyToClipboard
+      key="copy"
```

Comment:
I don't think you need a key here. Did you get a warning when omitting it ?

Reply:
Not needed, you're right. I just quickly made this change based on the existing code that could probably be cleaned up in a separate PR.